8Knot/pages/codebase/visualizations/cntrb_file_heatmap.py

```diff
@@ -203,10 +213,54 @@ def directory_dropdown(repo_id):
 
     # add top level directory to the list of directories
     directories.insert(0, "Top Level Directory")
+    logging.warning(f"DIRECTORY DROPDOWN - FINISHED")
 
     return directories, "Top Level Directory"
 
 
+def multi_query_helper(repos):
```

Comment:
for readability and consistency for helper functions in other visualizations move below create_figure

Reply:
++ updated